File: docs-examples/example-java/src/main/java/io/micronaut/configuration/openapi/docs/naming/Owner.java
package io.micronaut.configuration.openapi.docs.naming;

// tag::imports[]
import io.swagger.v3.oas.annotations.media.Schema;
// end::imports[]

// tag::clazz[]
class Owner {

    private Pet bird;
    private Pet cat;
    private Pet dog;

    @Schema(description = "Pet that is a bird") // <2>
    public Pet getBird() {
        return bird;
    }

    @Schema(description = "Pet that is a cat") // <3>
    public Pet getCat() {
        return cat;
    }

    @Schema(name = "Dog", description = "Pet that is a dog") // <4>
    public Pet getDog() {
        return dog;
    }
}
// end::clazz[]

File: docs-examples/example-kotlin/src/main/kotlin/io/micronaut/configuration/openapi/docs/naming/Owner.kt
package io.micronaut.configuration.openapi.docs.naming

// tag::imports[]
import io.swagger.v3.oas.annotations.media.Schema
// end::imports[]

// tag::clazz[]
class Owner {

    @get:Schema(description = "Pet that is a bird") // <2>
    var bird: Pet? = null

    @get:Schema(description = "Pet that is a cat") // <3>
    var cat: Pet? = null

    @get:Schema(name = "Dog", description = "Pet that is a dog") // <4>
    var dog: Pet? = null
}
// end::clazz[]
